frontend/__tests__/src/components/HomeGuide/HomeGuide.test.tsx

```diff
@@ -1,26 +1,80 @@
 import { HomeGuide } from '@src/components/HomeGuide'
-import { fireEvent, render } from '@testing-library/react'
+import { fireEvent, render, waitFor } from '@testing-library/react'
+import { setupStore } from '../../utils/setupStoreUtil'
+import { Provider } from 'react-redux'
+import { CREATE_NEW_PROJECT, IMPORT_PROJECT_FROM_FILE } from '../../fixtures'
+import userEvent from '@testing-library/user-event'
 
-const CREATE_NEW_PROJECT = 'Create a new project'
 const mockedUsedNavigate = jest.fn()
-
+const mockedUseAppDispatch = jest.fn()
 jest.mock('react-router-dom', () => ({
   ...jest.requireActual('react-router-dom'),
   useNavigate: () => mockedUsedNavigate,
 }))
 
+jest.mock('@src/hooks/useAppDispatch', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useAppDispatch: () => mockedUseAppDispatch,
+}))
+
+let store = setupStore()
+
+const setup = () => {
+  store = setupStore()
+  return render(
+    <Provider store={store}>
+      <HomeGuide />
+    </Provider>
+  )
+}
+beforeEach(() => {
```

Comment:
Before Each after Each is placed in describe

Reply:
yes!